example/etcd-backup-operator/backup_cr.yaml

```diff
@@ -10,3 +10,6 @@ spec:
     # e.g: "mybucket/etcd.backup"
     path: <full-s3-path>
     awsSecret: <aws-secret>
+    # leave blank to use aws, or add s3 compatible object store URL
+    # ie. endpoint: <tenancy>.compat.objectstorage.us-phoenix-1.oraclecloud.com
+    endpoint: <aws-endpoint>
```

Comment:
Please leave this example as it was to keep the doc simple now.

Reply:
does this really complicate it? I updated the instructions to sed an empty string as the value

Reply:
Let's just add the feature for this PR. We can care about examples & docs later. I have better ideas for it.